src/parser/jobs/sch/modules/DoTs.js

```diff
@@ -5,19 +5,25 @@ import {ActionLink, StatusLink} from 'components/ui/DbLink'
 import ACTIONS from 'data/ACTIONS'
 import STATUSES from 'data/STATUSES'
 import CoreDoTs from 'parser/core/modules/DoTs'
-import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import {TieredRule, TARGET, Requirement} from 'parser/core/modules/Checklist'
 import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
 import DISPLAY_ORDER from './DISPLAY_ORDER'
 
-// In ms
-const CLIPPING_SEVERITY = {
-	1000: SEVERITY.MINOR,
-	10000: SEVERITY.MEDIUM,
-	30000: SEVERITY.MAJOR,
+// In seconds
+const SEVERITIES = {
+	CLIPPING: {
+		6: SEVERITY.MINOR,
+		9: SEVERITY.MEDIUM,
+		12: SEVERITY.MAJOR,
+	},
+	UPTIME: {
+		84: TARGET.WARN,
+		91: TARGET.SUCCESS,
```

Comment:
I'm not a SCH by any stretch of the imagination, this feels a bit low to me for a DoT uptime target, especially one that isn't a comboed DoT. Am I thinking too much like a DPS main here?

Reply:
Well, for example, here's the uptime for some 99th and 100th percentiles in E1S/E2S:

* E1S
  * https://xivanalysis.com/analyse/1aAJLzymwHnpjBYF/2/3/
  * https://xivanalysis.com/analyse/hx1dtYCm4TMqA87G/7/8/
  * https://xivanalysis.com/analyse/NBR4GmL2PwxFch9Y/9/3/
* E2S
  * https://xivanalysis.com/analyse/ACq4jtrxGpJW92M3/11/3/
  * https://xivanalysis.com/analyse/G4XPKWgNnCZdDrh1/5/6/
  * https://xivanalysis.com/analyse/xpD6T3yFvrBnHLcA/14/6/

These seem to range from 80-odd to 96%. I mean, part of it could be the dps main mentality, but 95% already pretty hard to reach for even the highest damage SCHs on fflogs. Maybe change success to 95%, and warn from 85% and up?

Reply:
After some soul searching, I've decided to up the number. Given some uptime metrics I'm seeing from others, might be worth amending the text to talk about uptime detection with healers, but I see no reason that should block on the current percentages.